src/core/server/http/http_server.ts

```diff
@@ -55,6 +55,7 @@ export interface HttpServerSetup {
 
 export class HttpServer {
   private server?: Server;
+  private serverOptions?: ServerOptions;
```

Comment:
Is this needed still? I don't see any reference to it other than function it's assigned in. Were you planning on merging `start`'s config param with this one?